docs/api-docs/storefront/graphql/graphql-faceted-textual-search.md

```diff
@@ -0,0 +1,982 @@
+      maxPrice:200
+    },
+    rating:{
+      minRating:3,
+      maxRating:5
+    },
+    categoryEntityId:24,
+    searchSubCategories:false,
+    categoryEntityIds:[23],
+    brandEntityIds:[35],
+    productAttributes:[
+      {
+        attribute:"Color",
+        values:["Black"]
+      }
+    ],
+    isFreeShipping:true,
+    isFeatured:true,
+    isInStock:true
+  }
+  ... 
+) { 
+...
+```
+These filters affect **both** the products and facets that are returned. For example, filtering by rating returns only products that are within the specified rating range, as well as only facets that have products within the rating range. 
+
+See [GraphQL Playground](/graphql-playground) for descriptions of each filter. 
+
+## How to Get Products
+
+To get products, specify `products` as a field in `searchProducts`. Here is an example request that returns the first two products with a rating between three and five: 
+
+```GraphQL title="Request" lineNumbers
+query {
+  site {
+    search {
+      searchProducts(
+        filters: {
+          rating:{
+             minRating:3,
+             maxRating:5
+           }
+        }
+      ) {
+        products(first: 2) {
+          edges {
+            node {
+              entityId
+              name
+              prices {
+                price {
+                  value
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+```JSON title="Response" lineNumbers
+{
+  "data": {
+    "site": {
+      "search": {
+        "searchProducts": {
+          "products": {
+            "edges": [
+              {
+                "node": {
+                  "entityId": 80,
+                  "name": "[Sample] Orbit Terrarium - Large",
+                  "prices": {
+                    "price": {
+                      "value": 109
+                    }
+                  }
+                }
+              },
+              {
+                "node": {
+                  "entityId": 93,
+                  "name": "[Sample] 1 L Le Parfait Jar",
+                  "prices": {
+                    "price": {
+                      "value": 7
+                    }
+                  }
+                }
+              }
+            ]
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+You can sort the products that are returned using the `sort` field. Here is an example request that searches for products using a search term and sorts the products in alphabetical order:
+
+```GraphQL title="Request" lineNumbers
+query {
+  site {
+    search {
+      searchProducts(
+        filters: {
+          searchTerm: "Sample"
+        }
+        sort: A_TO_Z
+      ) {
+        products(first: 5) {
+          edges {
+            node {
+              entityId
+              name
+              prices {
+                price {
+                  value
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
```

Comment:
and here, `&nbsp;` if nesting not desired.

Reply:
Used new feature to nest while choosing the tab name